common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2921,6 +2921,29 @@
     ],
     "sqlState" : "42K09"
   },
+  "INVALID_VARIANT" : {
```

Comment:
shall we be more specific as this only happens when reading variant from parquet? something like `INVALID_VARIANT_FROM_PARQUET`

Reply:
Yes, currently only `parquet`, `json` and `binary` formats are supported. 
(PS: json - read & write, binary - read
We can be suffixed with 'FROM-PARQUET'. 
If we encounter `more general scenarios later`, we can refactor and reuse it.

Updated.